rfc6979/src/lib.rs

```diff
@@ -60,7 +60,7 @@ pub type ByteArray<Size> = GenericArray<u8, Size>;
 /// - `x`: secret key
 /// - `n`: field modulus
 /// - `h`: hash/digest of input message: must be reduced modulo `n` in advance
-/// - `data`: additional associated data, e.g. CSRNG output used as added entropy
+/// - `data`: personalization string, e.g. CSRNG output used as added entropy
```

Comment:
Though this name is fine for the parameter to `HmacDrbg::new`, RFC6979 does not refer to this parameter as "personalization string".

It's defined in [RFC6979 § 3.6](https://www.rfc-editor.org/rfc/rfc6979#section-3.6):

```
   o  Additional data may be added to the input of HMAC, concatenated
      after bits2octets(H(m)):

         K = HMAC_K(V || 0x00 || int2octets(x) || bits2octets(h1) || k')
```

RFC6979 uses the terminology "additional data" and `k'` to refer to it. That it's passed as the HMAC-DRBG personalization string is an implementation detail users of this API are unlikely to be aware of.

Reply:
Ack, thanks, removed this line from the PR.